flink/v1.12/flink/src/main/java/com/netease/arctic/flink/metric/MetricConstant.java

```diff
@@ -0,0 +1,17 @@
+package com.netease.arctic.flink.metric;
+
+/**
+ * metric constant
+ */
+public class MetricConstant {
+
+  /**
+   * The start time of arctic table's initialization when it used as build table in temporal join.
+   */
+  public static final String INIT_START_MS = "init-start-ms";
+  /**
+   * The end time of arctic table's initialization when it used as build table in temporal join.
+   */
+  public static final String INIT_END_MS = "init-end-ms";
```

Comment:
INIT_END_MS -> TEMPORAL_TABLE_INITIALIZE_ENDING_MS ?

Reply:
How about TEMPORAL_TABLE_INITIALISATION_END_TIMESTAMP

Reply:
It makes sense to me.